tests/console/perl_bootloader.pm

```diff
@@ -15,24 +14,49 @@ use strict;
 use warnings;
 use utils 'zypper_call';
 use power_action_utils 'power_action';
-use version_utils 'is_sle';
+use version_utils qw(is_sle is_opensuse);
 
 sub run {
     my ($self) = @_;
     select_serial_terminal;
+    zypper_call 'in perl-Bootloader';
```

Comment:
Is there a case where `pbl` is not pre-installed?

Reply:
Not that I know of, I thought it was better to make sure it's there before testing. I can remove it if not necessary.

Reply:
You could use `ensure_installed`

Reply:
So do you think it's not necessary to add this line?

Reply:
I thought that it should always come pre-installed, especially when the system uses GRUB2. If that is always true I would rather check the presence of the tool.